amgut/db/patches/0018.sql

```diff
@@ -0,0 +1,8 @@
+-- August 24, 2015
+-- add country column to zipcode table
+ALTER TABLE ag.zipcodes ADD COLUMN country varchar NOT NULL DEFAULT '';
+ALTER TABLE ag.zipcodes ALTER COLUMN country DROP DEFAULT;
+UPDATE ag.zipcodes SET country = 'United States' WHERE state IN ('AL','AK','AZ','AR','CA','CO','CT','DC','DE','FL','GA','HI','ID','IL','IN','IA','KS','KY','LA','ME','MD','MA','MI','MN','MS','MO','MT','NE','NV','NH','NJ','NM','NY','NC','ND','OH','OK','OR','PA','RI','SC','SD','TN','TX','UT','VT','VA','WA','WV','WI','WY');
+UPDATE ag.zipcodes SET country = 'Canada' WHERE state IN ('AB','BC','MB','NB','NL','NS','NT','NU','ON','PE','QC','SK','YT');
+-- Delete the rest so the table can repopulate correctly
```

Comment:
The table will repopulate as things are pulled down, so any zipcodes that are british, mexican, ukranian, etc participants will be repopulated in the DB as proper geocode info instead of the mostly empty info we have for these zipcodes right now. They were added by hand since the table had no way of populating itself.